### ISSUE TITLE
Error RUNNING HANDLER [enable microk8s addons]

### ISSUE THREAD

**Issue by peda** [z3dm4n/k8s-dev-cluster-hcloud#5]
When executing `make` the ansible script does fail for all nodes with the follwoing error:

```
failed: [n2] (item=dns) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "dns"], "delta": "0:00:00.438042", "end": "2021-01-21 10:38:17.716914", "item": "dns", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:17.278872", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n3] (item=dns) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "dns"], "delta": "0:00:00.486597", "end": "2021-01-21 10:38:17.757943", "item": "dns", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:17.271346", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n1] (item=dns) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "dns"], "delta": "0:00:00.532653", "end": "2021-01-21 10:38:17.806155", "item": "dns", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:17.273502", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n2] (item=helm3) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "helm3"], "delta": "0:00:00.445774", "end": "2021-01-21 10:38:19.006074", "item": "helm3", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:18.560300", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n3] (item=helm3) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "helm3"], "delta": "0:00:00.476856", "end": "2021-01-21 10:38:19.028085", "item": "helm3", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:18.551229", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n1] (item=helm3) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "helm3"], "delta": "0:00:00.449568", "end": "2021-01-21 10:38:19.046184", "item": "helm3", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:18.596616", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n2] (item=ingress) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "ingress"], "delta": "0:00:00.419680", "end": "2021-01-21 10:38:20.262209", "item": "ingress", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:19.842529", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n1] (item=ingress) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "ingress"], "delta": "0:00:00.413806", "end": "2021-01-21 10:38:20.287827", "item": "ingress", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:19.874021", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n3] (item=ingress) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "ingress"], "delta": "0:00:00.444917", "end": "2021-01-21 10:38:20.293772", "item": "ingress", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:19.848855", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n1] (item=storage) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "storage"], "delta": "0:00:00.415779", "end": "2021-01-21 10:38:21.518862", "item": "storage", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:21.103083", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n3] (item=storage) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "storage"], "delta": "0:00:00.438202", "end": "2021-01-21 10:38:21.571344", "item": "storage", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:21.133142", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
failed: [n2] (item=storage) => {"ansible_loop_var": "item", "changed": true, "cmd": ["/snap/bin/microk8s.enable", "storage"], "delta": "0:00:00.480774", "end": "2021-01-21 10:38:21.585400", "item": "storage", "msg": "non-zero return code", "rc": 1, "start": "2021-01-21 10:38:21.104626", "stderr": "Traceback (most recent call last):\n  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>\n    enable(prog_name='microk8s enable')\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__\n    return self.main(*args, **kwargs)\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main\n    _verify_python3_env()\n  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env\n    'mitigation steps.' + extra)\nRuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.\n\nThis system supports the C.UTF-8 locale which is recommended.\nYou might be able to resolve your issue by exporting the\nfollowing environment variables:\n\n    export LC_ALL=C.UTF-8\n    export LANG=C.UTF-8", "stderr_lines": ["Traceback (most recent call last):", "  File \"/snap/microk8s/1910/scripts/wrappers/enable.py\", line 43, in <module>", "    enable(prog_name='microk8s enable')", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 716, in __call__", "    return self.main(*args, **kwargs)", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/core.py\", line 675, in main", "    _verify_python3_env()", "  File \"/snap/microk8s/1910/usr/lib/python3/dist-packages/click/_unicodefun.py\", line 119, in _verify_python3_env", "    'mitigation steps.' + extra)", "RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Either run this under Python 2 or consult http://click.pocoo.org/python3/ for mitigation steps.", "", "This system supports the C.UTF-8 locale which is recommended.", "You might be able to resolve your issue by exporting the", "following environment variables:", "", "    export LC_ALL=C.UTF-8", "    export LANG=C.UTF-8"], "stdout": "", "stdout_lines": []}
```

**Comment by peda** [comment]
adding the following to lines to the `snap.enable` tasks in the `01` and `04` file seem to solve the issue for me

```
        environment:
            LC_ALL: "C.UTF-8"
            LANG: "C.UTF-8"
```

**Comment by z3dm4n** [comment]
Thanks for reporting!

Although I was not able to reproduce this issue with `ansible 2.9.6`, I added the environment variables as they won't do any harm and ensure better portability. 

Fixed in commit 2e77ff0c6e4c8677b17231d053baa017105b8be5. 

You may close this issue, if this solves your problem.

**Comment by z3dm4n** [comment]
Closed due to inactivity.
